tests/components/dynalite/test_config_flow.py

```diff
@@ -70,21 +72,27 @@
 async def test_existing_update(hass):
     """Test when the entry exists with the same config."""
     host = "1.2.3.4"
-    mock_entry = MockConfigEntry(
-        domain=dynalite.DOMAIN, unique_id=host, data={dynalite.CONF_HOST: host}
-    )
-    mock_entry.add_to_hass(hass)
     with patch(
         "homeassistant.components.dynalite.bridge.DynaliteDevices.async_setup",
         return_value=True,
     ), patch(
         "homeassistant.components.dynalite.bridge.DynaliteDevices.available", True
     ):
+        assert await hass.config_entries.flow.async_init(
+            dynalite.DOMAIN,
+            context={"source": config_entries.SOURCE_IMPORT},
+            data={dynalite.CONF_HOST: host},
```

Comment:
Looking at the config flow -> we shouldn't have used host as unique ID. It's not unique because IPs can change. Is there no real unique ID available?

Reply:
no. the problem is that the gateway is "dumb" in that it only sends messages to the network that doesn't have any unique IDs. Just 255 areas, with their presets and channels. 
However, I don't think it is a big issue since the gateway is so "unprogrammable" that you usually need to invite the integrator who installed it to reprogram it, so I don't think it is something that actually changes in real life. 
I went over the Dynalite documentation again, and couldn't find anything that would resemble a unique ID in a better way that IP addresses, but open to suggestions.

Reply:
If the ip address changes, the config entry would need to be removed anyway, since the integration would stop working otherwise.

For config entries unique_id, I think ip address is ok.

Reply:
If we don't have a unique ID, we shouldn't make one up. We shouldn't set a unique ID to a non-unique value.

Reply:
Host is unique for the existence of the config entry.

Reply:
I can think of a bad case which is if there are more than one entry and they swap addresses with each other.

So yeah, that case is real and not good.

Reply:
i could use the bridge name optional argument, but then it depends on the user not changing it. Is this preferable?
Alternatively, I wasn't aware that there is an option without a unique ID. Is there some documentation for it?

Reply:
BTW, when you look at these gateways in a real life Dynalite, they are simply the network address that connects the Dynet physical network to an IP network, so I cannot imagine a real case where there will be more than one network. However, I could be wrong, maybe for very large installations.

Reply:
in any case, would appreciate whatever guidance works best here and I will implement it

Reply:
maybe if i move away from the unique_id pattern and simply check when I import from configuration.yaml if the config has changed and if it did, update the bridge?
It is pretty much the same flow that I have now but would perhaps be better designed?

Reply:
If we don't use unique_id, we instead need to check that we don't set up the same host more than once. It's basically what the core does when we use the unique_id api, but without any further consequences.

We can still use the update listener on config entry change also without config entry unique_id. If the configuration.yaml has changed we update the corresponding config entry which will update the bridge via the listener.

Reply:
ok. implemented it without unique_id